fe1-web/src/core/network/__tests__/NetworkManager.test.ts

```diff
@@ -3,11 +3,18 @@ import { expect } from '@jest/globals';
 import { NetInfoState } from '@react-native-community/netinfo';
 
 import { getNetworkManager } from '../NetworkManager';
+import { SendingStrategy } from '../strategies/ClientMultipleServerStrategy';
 import { mockJsonRpcPayload } from './utils';
 
 jest.mock('websocket');
 
 const networkManager = getNetworkManager();
+type NetworkManagerType = typeof networkManager;
+
+const {
+  getMockNetworkManager,
+}: { getMockNetworkManager: (sendingStrategy: SendingStrategy) => NetworkManagerType } =
+  jest.requireMock('core/network/NetworkManager.ts');
```

Comment:
the spacing / newlines here + complex types make this very confusing / hard to read.
try to make this more readable.